app/models/waste_exemptions_engine/registration_exemption.rb

```diff
@@ -16,7 +18,8 @@ class RegistrationExemption < ActiveRecord::Base
         .order(:registered_on, :registration_id)
     }
 
-    scope :not_expired, -> { where("expires_on > CURRENT_DATE") }
+    scope :not_expired, -> { where("expires_on >= CURRENT_DATE") }
+    scope :expired, -> { where("expires_on < CURRENT_DATE") }
```

Comment:
I'm surprised this doesn't clash with any scoping already provided by AASM, since `expired` is also the name of a state. I think something like `should_be_expired` or `past_expiry_date` might be clearer that it's not based on the actual state of the exemption.

Reply:
Also, do we need any tests for these? Maybe overkill but let me know what you think.

Reply:
Well, the scope from AASM is defined above, hence this will take precedence. I can add test coverage but I am quite sure. Maybe better to name this differently?

Reply:
On second thoughts, I think is totally fine that we rename it since the status change happens in a background job. It does mean that we will never check the "state" but rather the date, and that's totally fine. Probably then we should move this scope in the same concern that deals with adding AASM. Let me know what you think :)

Reply:
@cintamani Sounds like a good spot for it to me 👍

Reply:
I was actually suggesting not to rename it and move it in the AASM file for more clarity around the fact that we are overriding it :) Shall we have a call and discuss it later? :)